src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1142,7 +1159,21 @@ private void modifyCollaborators(@NonNull Collection<GHUser> users,
         }
 
         // Make sure that the users collection doesn't have any duplicates
-        for (GHUser user : new LinkedHashSet<GHUser>(users)) {
+        for (GHUser user : new LinkedHashSet<>(users)) {
```

Comment:
This version of `modifyCollaborators` should be removed.

Reply:
like outright deleted? isn't that a breaking change?

Reply:
@jgangemi  Still need to do this.

Reply:
@bitwiseman fixed